src/fidesops/service/privacy_request/request_service.py

```diff
@@ -0,0 +1,73 @@
+import logging
+from datetime import datetime
+from typing import Optional, Any, Dict, Set, List
+
+from sqlalchemy.orm import Session
+
+from fidesops.models.policy import Policy, ActionType
+from fidesops.models.privacy_request import PrivacyRequestStatus, PrivacyRequest
+from fidesops.schemas.drp_privacy_request import DrpPrivacyRequestCreate
+from fidesops.schemas.masking.masking_configuration import MaskingConfiguration
+from fidesops.schemas.masking.masking_secrets import MaskingSecretCache
+from fidesops.schemas.policy import Rule
+from fidesops.schemas.redis_cache import PrivacyRequestIdentity
+from fidesops.service.masking.strategy.masking_strategy_factory import get_strategy
+
+logger = logging.getLogger(__name__)
+
+
+def retrieve_policy(db: Session, field: str, value: Any) -> Optional[Policy]:
+    return Policy.get_by(
+        db=db,
+        field=field,
+        value=value,
+    )
+
+
+def build_required_privacy_request_kwargs(
+    requested_at: Optional[datetime], policy_id: str
+) -> Dict[str, Any]:
+    return {
+        "requested_at": requested_at,
+        "policy_id": policy_id,
+        "status": PrivacyRequestStatus.pending.value,
+    }
+
+
+def create_privacy_request(db: Session, kwargs: Dict[str, Any]):
+    return PrivacyRequest.create(db=db, data=kwargs)
```

Comment:
ideally let service handle calls to db. Better separation of concerns imo

Reply:
I'm not sure these are adding any value beyond being housed here? What's the justification for this being better than using the ORM util methods?

Reply:
I went ahead and removed the db abstraction for now. Imo, endpoint/controller logic shouldn't be mixed with db calls, maybe a topic of discussion for later. It can reduce chances of duplicate code / circular dependencies in the future. Say, for example, we wish to set a log status whenever we create a privacy request. If we directly called db from the endpoint, we'd have to update every instance of it, vs updating once in the service

Reply:
Thanks! You're right, I think that's useful. My only concern is that we create these methods now and those conditions never manifest, or they do and somebody decides to go a different way.